babeltrace-2.yaml

```diff
@@ -0,0 +1,119 @@
+package:
+  name: babeltrace-2
+  version: 2.1.0
+  epoch: 0
+  description: "An open-source trace manipulation toolkit."
+  copyright:
+    - license: MIT
```

Comment:
Do we need to list multiple licenses here?
 - https://github.com/efficios/babeltrace/blob/master/LICENSE
 
 This may require a bit of reading / cross-checking